disaster_recovery/restore/disaster_recovery.py

```diff
@@ -0,0 +1,621 @@
+        self.role = 'breakglass'
+        self.role_to_assume = str(f'arn:aws:iam::{self.account}:role/{self.role}')
+        self.backup_role_to_assume = str(f'arn:aws:iam::{self.backup_account}:role/{self.role}')
+        self.region = 'eu-west-1'
+        self.client = None
+        self.client_kms = None
+        self.client_backup_rds = None
+        self.client_backup_kms = None
+        self.restore_from_remote = restore_from_remote
+        self.example_var = None
+        self.instances = []
+        self.db_cluster_identifier_source = cluster_from
+        self.db_cluster_identifier_target = self.db_cluster_identifier_source if cluster_to is None else cluster_to
+        self.same_target = True if self.db_cluster_identifier_target == self.db_cluster_identifier_source else False
+        self.SnapshotIdentifier = snapshot_id
+        self.multi_az_override = multi_az_override
+        self.MultiAZ = False
+        self.engine_mode_override = engine_mode_override
+        self.EngineMode = None
+        self.PointInTimeRecovery = datetime.strptime(pitr, '%Y-%m-%d %H:%M:%S')
+        self.AllocatedStorage = 1
+        self.AvailabilityZones = ['eu-west-1b', 'eu-west-1a', 'eu-west-1c']
+        self.BackupRetentionPeriod = 14
+        self.DBClusterParameterGroup = 'default.aurora-postgresql10'
+        self.DBSubnetGroup = 'private'
+        self.Engine = 'aurora-postgresql'
+        self.EngineVersion = '10.18'
+        self.VpcSecurityGroups = None
+        self.StorageEncrypted = True
+        self.KmsKeyId = None
+        self.Capacity = None
+        self.EngineMode = 'serverless'
+        self.HttpEndpointEnabled = False
+        self.DeletionProtection = True
+        self.TagList = [{
+            'Key': 'owner',
+            'Value': 'OPG Supervision'
+        }, {
+            'Key': 'environment-name',
+            'Value': 'development'
+        }, {
+            'Key': 'application',
+            'Value': 'Digideps'
+        }, {
+            'Key': 'is-production',
+            'Value': '0'
+        }, {
+            'Key': 'business-unit',
+            'Value': 'OPG'
+        }, {
+            'Key': 'infrastructure-support',
+            'Value': 'OPG WebOps: opgteam@digital.justice.gov.uk'
+        }]
+        self.AutoMinorVersionUpgrade = True
+
+    def restore(self):
+        if self.account is None:
+            print('No valid account set! Exiting...')
+            os.exit(1)
+        self.create_digideps_client_session()
+
+        if self.db_cluster_identifier_source is not None:
+            self.get_cluster_info()
+            if self.EngineMode != 'serverless':
+                self.get_instance_info()
+            if self.db_cluster_identifier_source == self.db_cluster_identifier_target and self.DeletionProtection:
+                self.drop_protection()
+
+        self.apply_overrides()
+
+        if self.PointInTimeRecovery != datetime.strptime('1900-01-01 00:00:00', '%Y-%m-%d %H:%M:%S'):
```

Comment:
Assuming this is the format `self.PointInTimeRecovery` must be set to in order to restore to a point in time we could add a little more context here by extracting `'1900-01-01 00:00:00'` to a var with a name describing its intent (something like `requiredDateTimeFormat`?)

Reply:
Good call